### ISSUE TITLE
Student/Instructor: help page: add navbar

### ISSUE THREAD

**Issue by craaaa** [TEAMMATES/teammates#8227]
**Description of feature/enhancement**
It is currently only possible to return to the home page when using the navbar on `instructorHelp.jsp` and `studentHelp.jsp` as shown below. It would be good to include the navigation bar of the static site (e.g. Contact Us/About Us etc) and in particular the login links.

![screenshot from 2017-12-20 23-32-57](https://user-images.githubusercontent.com/12183503/34215151-79a8ec88-e5df-11e7-8553-1ddd88a1b15b.png)

**Justification**
Rather than having to return to the home page first, users of the site will be able to more easily log in and access their evaluations if the login links are immediately available. Adding the additional navbar features will also help to prevent these pages from being 'stranded' from the rest of the site when users have finished reading the information they were looking for, as they can easily navigate away.

**Comment by deydebaditya** [comment]
Fixed this in PR #8229 
Please do review the PR, and tell if any changes are necessary.

**Comment by damithc** [comment]
The help page is only available from within the app (it is considered part of the app rather than part of the static site) and it opens in a new tab/window (this is to allow user to refer to instructions in parallel to using the app). Hence there is no need to 'go back'?

**Comment by deydebaditya** [comment]
@damithc yes, that'd be true. On the other hand, I guess there would be users who would like to navigate directly to the login or to other static pages after reading the help, adding a navigation wouldn't hurt, rather I think it would improve the overall user experience. I would love reviews and criticism on this, it is just a view from my side.

**Comment by LiHaoTan** [comment]
@deydebaditya the static pages (i.e. the homepage and others) and the web application are distinct. The source of confusion probably comes from the navbars sharing the same design. Perhaps the navbar design can be altered for the static pages.

It won't quite make sense for the help page's navbar to link back to other static pages.

**Comment by deydebaditya** [comment]
Okay. I get what you're saying. Will take a note of this. Thanks. :)

**Comment by craaaa** [comment]
Oops, sorry for the confusion! I actually got to `instructorHelp.jsp` through the static pages by following the following steps:
`Home page` > `Features static page` > `Help page link in first line`
so thought that instructorHelp was part of the static site.

**Comment by LiHaoTan** [comment]
@craaaa ah I didn't notice the `Features` page has a link to the help page as well. I don't believe either of you is wrong, and there is definitely room for improvement in the current design.

Some points of the current design to explore:
- The instructor and student help pages are really part of the web application, but the instructor help page is also linked from the `Features` page. Since the help pages are shared, some improvements may possibly be made in distinguishing the project site and the web application.
- There are two references to `instructorHelp.jsp` which one opens in a new tab/window and the other does not.
- The main site's navbar looks too similar to the navbar in the web application.

**Comment by damithc** [comment]
How about we make all 'help page' links in the features page open in new window?
And yes, we can make the static site and app slightly different (but not too different). 
The latter can be a separate issue.

**Comment by wkurniawan07** [comment]
Resolved in V7.
